### ISSUE TITLE
v1 mac build fail (thread priority update)

### ISSUE THREAD

**Issue by lnikj** [VCVRack/Rack#1190]
6700K; 395X; 10.13.6
Apple LLVM version 10.0.0 (clang-1000.11.45.5)
Target: x86_64-apple-darwin17.7.0

Commit: 6986801ddd434a325b9621255ddd1da3c3b4ecd4

```
src/system.cpp:93:2: error: no matching function for call to 'pthread_setname_np'
        pthread_setname_np(pthread_self(), name.c_str());
        ^~~~~~~~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include/pthread.h:513:5: note:
      candidate function not viable: requires 1 argument, but 2 were provided
int     pthread_setname_np(const char*);
```

**Comment by AndrewBelt** [comment]
Fixed in a97ae1c

**Comment by modlfo** [comment]
Great! I just hit the same issue a few minutes ago.
